pyrasterframes/src/main/python/docs/reference.pymd

```diff
@@ -126,26 +126,29 @@ Functions to create a new Tile column, either from scratch or from existing data
 ### rf_make_zeros_tile
 
 ```
-Tile rf_make_zeros_tile(Int tile_columns, Int tile_rows, String cell_type_name)
+Tile rf_make_zeros_tile(Int tile_columns, Int tile_rows, [CellType cell_type])
+Tile rf_make_zeros_tile(Int tile_columns, Int tile_rows, [String cell_type_name])
```

Comment:
@vpipkt Sometime before 1.0, we should change these signatures to use the `var: Type` ordering, as that's what type hints in Python 3.7 use (along with Scala).

That said, really want to get this stuff generated by Sphinx.